### PR TITLE
FIX: don't validate and render the polls inside a quoted post.

### PR DESCRIPTION
**Pull request by vinothkannans**
Previously, we had issues while approving the posts since we validated the polls inside quotes.

### CODE REVIEW

File: plugins/poll/assets/javascripts/initializers/extend-for-poll.js [hunk 1]
```diff
@@ -69,7 +69,9 @@ function initializePolls(api) {
   });
 
   function attachPolls($elem, helper) {
-    const $polls = $(".poll", $elem);
+    const $polls = $(".poll", $elem).filter(function () {
```
Comment (jjaffeux) [inline plugins/poll/assets/javascripts/initializers/extend-for-poll.js:72]: please take every opportunity to remove jquery usage

**Review by jjaffeux — CHANGES_REQUESTED** [review]

**Comment by jjaffeux** [comment]
There are also linting errors to fix.

**Comment by SamSaffron** [comment]
This can wait till 2.9

@vinothkannans can you rebase.

**Comment by SamSaffron** [comment]
@vinothkannans can you finish this one off? Looks like all that is left is to change to use https://developer.mozilla.org/en-US/docs/Web/API/Element/getElementsByClassName and we can merge this in.

**Review by jjaffeux — APPROVED** [review]

**Pull request merged**